axelar-chains-config/info/mainnet.json

```diff
@@ -1221,7 +1230,8 @@
       "staticGasOptions": {
         "gasLimit": 100000000,
         "maxFeePerGas": 1000000000
-      }
+      },
+      "finality": 60
```

Comment:
we're using `100` for confirmation height
https://lcd-axelar.imperator.co/axelar/evm/v1beta1/confirmation_height/filecoin
https://lcd-axelar.imperator.co/static/swagger/#/QueryService/ConfirmationHeight

If chain repo has an incorrect value, you can update it there